libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/ProgressDialog.kt

```diff
@@ -81,6 +95,8 @@ sealed interface ProgressDialogType {
 private fun ProgressDialogContent(
     modifier: Modifier = Modifier,
     text: String? = null,
+    isCancellable: Boolean = true,
```

Comment:
Maybe set to false by default like the public fun?

Reply:
yes good catch.